docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -13,16 +13,89 @@ weight: 850
+1. Go to the dashboard **Settings** and add a new variable called `traceId`, of variable type **Custom**, giving it a label if required. Select **Apply** to add the variable to the dashboard. <br /> {{< figure  src="/static/img/docs/panels/traces/screenshot-traces-custom-variable.png" maxWidth="25%" caption="Add a Custom variable" >}}
+
+1. Verify that the panel works by using a valid trace ID for the data source used for the trace panel and editing the ID in the dashboard variable. <br /> {{< figure src="/static/img/docs/panels/traces/screenshot-traces-traceid-panel.png" caption="Results of query in Trace panel" >}}
+
+## Add TraceQL with table visualizations
+
+While you can add a trace panel visualization to a dashboard, having to manually add trace IDs as a dashboard variable is cumbersome.
+It’s more useful to instead be able to use TraceQL queries to search for specific types of traces and then select appropriate traces from matching results.
+
+1. In the same dashboard that we added the trace visualization, select **Add panel** to add a new visualization panel.
+1. Select the same trace data source you used in the previous section.
+1. In the panel search pane, select the Visualizations tab and search for **Table**.
+1. Select the **TraceQL** tab from the query editor.
+1. On the **Panel editor**, enter a **Title** for your trace panel.
+1. Add an appropriate TraceQL query to search for traces that you would like to visualize in the dashboard. We’ll use a simple, static query as an example. You can write the TraceQL query as a template query to take advantage of other dashboard variables, if they exist. This lets you create dynamic queries based on these variables. <br /> {{< figure src="/static/img/docs/panels/traces/screenshot-traces-dynamic-query.png" caption="Create a dynamic query" >}}
+
+When results are returned from a query, the results are rendered in the panel’s table.
+
+{{< figure src="/static/img/docs/panels/traces/screenshot-traces-returned-query.png" caption="Results of a returned query in the panel table" >}}
+
+### Use a variable to add other links to traces
+
+Traces results include links to an Explorer page that render the trace. You can add other links to traces in the table that fill in the `traceId` dashboard variable when selected so that the trace is visualized in the same dashboard.
+
+To do so, we’re going to create a set of data links in the panel.
+
+1. Select **Add link** from the **Data links** section of the right menu.
+1. Add a **Title** for the data link.
+1. Find the UUID of the dashboard by looking in your browser’s address bar when the full dashboard is being rendered. Because this is a link to a dashboard in the same Grafana instance, only the path of the dashboard is required. <br /> {{< figure src="/static/img/docs/panels/traces/screensnot-traces-uuid-url.png" caption="Unique identifier for the dashboard" >}}
```

Comment:
```suggestion
1. Find the UUID of the dashboard by looking in your browser’s address bar when the full dashboard is being rendered. Because this is a link to a dashboard in the same Grafana stack, only the path of the dashboard is required. <br /> {{< figure src="/static/img/docs/panels/traces/screensnot-traces-uuid-url.png" caption="Unique identifier for the dashboard" >}}
```